content/workshops/How to be a great Mentor/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: How To Be A Great Mentor
+---
+
+Link to the slides: https://docs.google.com/presentation/d/1rcw9ZIgt392DVsRY_jq1uUl6LjedfAtufM6qMkfz4fA/edit#slide=id.g62442b9240_0_1 
```

Comment:
Can we embed the slides? refer to page: https://umuzi-org.github.io/tech-department/workshops/agile-scrum/

I made a copy in the tech dept drive, all presentations should be in a shared location, you can use this link: https://docs.google.com/presentation/d/1LkIo9YlKi74hM_OaKB-u01X3GfkgNc_mKwjl0m-KtRk/edit#slide=id.p